File: src/main/java/com/woomulwoomul/woomulwoomulbackend/domain/base/ServiceStatus.java
package com.woomulwoomul.woomulwoomulbackend.domain.base;

import lombok.Getter;

@Getter
public enum ServiceStatus {
    ACTIVE("활성화"),
    USER_DEL("사용자 삭제"),
    ADMIN_DEL("관리자 삭제"),
    ;

    private final String text;

    ServiceStatus(String text) {
        this.text = text;
    }
}

File: src/main/kotlin/com/woomulwoomul/woomulwoomulbackend/domain/base/ServiceStatus.kt
package com.woomulwoomul.woomulwoomulbackend.domain.base

enum class ServiceStatus(
    val text: String,
) {
    ACTIVE("활성화"),
    USER_DEL("사용자 삭제"),
    ADMIN_DEL("관리자 삭제"),
}